src/status_im/contexts/preview/quo/selectors/react.cljs

```diff
@@ -37,40 +37,39 @@
 
 (defn preview-react
   []
-  (let [state             (reagent/atom {:add-reaction? true
+  (let [[state set-state] (rn/use-state {:add-reaction? true
                                          :max-count     1000
                                          :reaction-ids  [1 2 3]
                                          :use-case      :default})
-        pressed-reactions (reagent/atom #{1})]
-
-    (fn []
-      (let [reactions (mapv (fn [reaction-id]
-                              {:emoji-reaction-id reaction-id
-                               :emoji-id          reaction-id
-                               :emoji             (get constants/reactions reaction-id)
-                               :quantity          (memo-gen-quantity (:max-count @state) reaction-id)
-                               :own               (contains? @pressed-reactions reaction-id)})
-                            (:reaction-ids @state))]
-        [preview/preview-container
-         {:state      state
-          :descriptor descriptor}
-         [rn/view
-          {:padding-bottom     150
-           :padding-vertical   60
-           :padding-horizontal 20
-           :border-radius      16
-           :background-color   (when (= :pinned (:use-case @state))
-                                 (colors/custom-color :blue 50 10))
-           :align-items        :flex-start}
-          [quo/react
-           {:reactions     reactions
-            :add-reaction? (:add-reaction? @state)
-            :use-case      (:use-case @state)
-            :on-press      (fn [reaction]
-                             (let [reaction-id    (:emoji-id reaction)
-                                   change-pressed (partial swap! pressed-reactions)]
-                               (if (contains? @pressed-reactions reaction-id)
-                                 (change-pressed disj reaction-id)
-                                 (change-pressed conj reaction-id))))
-            :on-long-press identity
-            :on-press-new  identity}]]]))))
+        pressed-reactions (reagent/atom #{1})
```

Comment:
Can we remove the ratom here for consistency?

Reply:
Sure, I'll do that